catalog/core/catalog-core-standardframework/pom.xml

```diff
@@ -335,6 +340,7 @@
                             ddf.catalog.content.operation;version="2.0",
                             ddf.catalog.content.plugin;version="2.0",
                             ddf.catalog.content;version="2.0",
+                            ddf.catalog.core.versioning,
```

Comment:
If you don't supply one it will use ${project.version} which means it will change with every release (even though the actual interface did not change).  I would follow the same pattern as the other exports.